cmd/osv-scanner/fix/main.go

```diff
@@ -160,6 +161,11 @@ func Command(stdout, stderr io.Writer, r *reporter.Reporter) *cli.Command {
 				Name:     "ignore-dev",
 				Usage:    "ignore vulnerabilities affecting only development dependencies",
 			},
+			&cli.BoolFlag{
+				Category: vulnCategory,
+				Name:     "maven-ignore-management",
```

Comment:
should we invert this to `maven-fix-management` ? and make the default ignore these?

Reply:
Alternatively, should we generalize this to `--groups` instead as we discussed previously?

Reply:
discussed offline re `--group` -- keeping this flag (but inverted) SGTM.